### PR TITLE
Skillonomy 102 new report

### PR DESCRIPTION
**Pull request by dyudyunov**
**WORK IN PROGRES! DO NOT MERGE!**
**Description:** New report for users of current course

**Youtrack:** https://youtrack.raccoongang.com/issue/SKILLONOMY-102

**Reviewers:**
- [@oleksandra-martyntseva  ] 
- [@RuslanSkiraRaccoongang  ]

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation in source code updated
- [ ] All related Confluence documentation is updated (including deployment documentation)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by raccoondeploy** [comment]
retest this PR please

**Comment by raccoondeploy** [comment]
retest this PR please

**Comment by raccoondeploy** [comment]
retest this PR please

File: lms/djangoapps/branding/views.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ def index(request):
                 'ALWAYS_REDIRECT_HOMEPAGE_TO_DASHBOARD_FOR_AUTHENTICATED_USER',
                 settings.FEATURES.get('ALWAYS_REDIRECT_HOMEPAGE_TO_DASHBOARD_FOR_AUTHENTICATED_USER', True)):
             return redirect(reverse('dashboard'))
-
-    if settings.FEATURES.get('AUTH_USE_CERTIFICATES'):
+
```
Comment (dyudyunov) [inline lms/djangoapps/branding/views.py]: already corrected, it will not be in next commit

**Pull request closed without merging**